scripts/ovpsim_log_to_trace_csv.py

```diff
@@ -113,7 +117,7 @@ def process_ovpsim_sim_log(ovpsim_log, csv, full_trace = 0):
         prev_trace.binary = trace_bin
         prev_trace.addr = trace_addr
         #prev_trace.privileged_mode = trace_mode
-        prev_trace.instr = trace_instr
+        prev_trace.instr = trace_instr # wrong
```

Comment:
What's the meaning of this comment?

Reply:
I feel there is some logic wrong in this section and this line '# wrong' might be superflous - but until we have many more regression tests I am just highlighting that this needs looking at in future maybe it needs a TODO - to investigate not 'wrong' - pls just decde

Reply:
OK, got it. I have fixed a few issues over the last couple of days. You can try to simulate with top of the tree to see if the issue is gone.